modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -5658,6 +5718,7 @@ private void _updateLocalizedObjectEntryValues(
 	@Inject
 	private ObjectFilterLocalService _objectFilterLocalService;
 
+	private String _objectRelationship1Name;
```

Comment:
You can rename it to `_objectRelationshipName`, since there's no `_objectRelationship2Name`

Reply:
Done!